README.md

```diff
@@ -50,11 +50,12 @@ The [export](https://huggingface.co/docs/optimum/exporters/overview) and optimiz
 | Features                           | ONNX Runtime       | Neural Compressor  | OpenVINO           | TensorFlow Lite    |
 |:----------------------------------:|:------------------:|:------------------:|:------------------:|:------------------:|
 | Graph optimization                 | :heavy_check_mark: | N/A                | :heavy_check_mark: | N/A                |
-| Post-training Dynamic Quantization | :heavy_check_mark: | :heavy_check_mark: | N/A                | :heavy_check_mark: |
-| Post-training Static Quantization  | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Dynamic Quantization               | :heavy_check_mark: | :heavy_check_mark: | N/A                | :heavy_check_mark: |
+| Static Quantization                | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
```

Comment:
I think it's better as it was - you have QAT/PTQ are distinct than dynamic/static

Reply:
Updated in [`e2d018`](https://github.com/huggingface/optimum/pull/942/commits/e2d018ddb7d505ef7ff0ce51a92f0693bf8bf8a8)